src/main/scala/org/dia/apps/AccumulationsApp.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dia.apps
+
+import java.io.{ File, PrintWriter }
+
+import scala.collection.mutable
+import scala.io.Source
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.security.UserGroupInformation
+
+import org.apache.spark.rdd.RDD
+
+import org.dia.core.{SciDataset, SciSparkContext, SRDDFunctions, Variable}
+import org.dia.urlgenerators.OpenDapTRMMURLGenerator
+import org.dia.utils.{FileUtils, NetCDFUtils}
+
+object AccumulationsApp extends App {
+  val logger = org.slf4j.LoggerFactory.getLogger(this.getClass)
+  val masterURL = if (args.isEmpty) "local[*]" else args(0)
+  val partitions = if (args.length <= 1) 8 else args(1).toInt
+  val hdfsURL = if (args.length <= 2) "hdfs://localhost:9000" else args(2)
+  val varname = if (args.length <= 3) "precipitation" else args(3)
+  val outputLoc = if (args.length <= 4) "output" else args(4)
+  val credentialsFilePath = "src/test/resources/TestHTTPCredentials"
+  val credentialList = Source.fromFile(credentialsFilePath)
+    .getLines()
+    .map(p => {
+      val split = p.split("\\s+")
+      (split(0), split(1))
+    }).toList
+  val username = credentialList(0)._2
```

Comment:
@kwhitehall 
You may not need the above boiler plate code.
Consider the following lines in SciSparkContextTest :
https://github.com/SciSpark/SciSpark/blob/eaad943559f91eae9ec702e3da0fde9ffe2862b8/src/test/scala/org/dia/core/SciSparkContextTest.scala#L32-L33

Of course, this only pulls credentials from our file and is not general enough for user provided credentials files. What do you think? It could just be another issue. That is create a higher level function that reads in credential files.